docs/faq/admin.txt

```diff
@@ -97,8 +97,8 @@ like to make should be possible by editing the stylesheet.
 What browsers are supported for using the admin?
 ================================================
 
-The admin provides a fully-functional experience to `YUI's A-grade`_ browsers,
-with the notable exception of IE6, which is not supported.
+The admin provides a fully-functional experience to most of the `YUI's A-grade`_ browsers,
+with some exceptions, i.e, IE6, which is not supported.
```

Comment:
I think we should be more specific here adding "most of the" is not enough, IMO. Also "YUI's A-grade" is quite outdated.

Reply:
So should we create a list of browsers that supports admin?  Do you have any suggestions?

Reply:
We should probably find a better reference (because `yui` is outdated) and explain our policy instead of creating a list of browsers (see [discussion](https://groups.google.com/forum/#!msg/django-developers/MN9nz5rJODY/MAIOEVKCAQAJ)). For example, we've dropped support for Internet Explorer 8 an below in Django 1.9.

Reply:
It seems that we dropped support for IE9 in ba83378a7762c51be235b521aa5b48233d6c6c82.

Reply:
I've read the discussion. I understand. What is the policy? By the way thanks for quick answers.

Reply:
I don't think there is a policy. The ticket, in my reading, seems to be to make one that is more in line with the code.

Personally, guided by https://gs.statcounter.com/browser-version-partially-combined-market-share/desktop/worldwide/#monthly-201811-201911-bar - it looks like supporting anything older than IE11 wouldn't be a good use of time.

I'm in favour of being aggressive here, as this would allow us to modernise some of the javascript components in the admin, or perhaps even replace them with native html5 input types so as to not re-invent the wheel, and to shift the maintenance burden from Django onto browser vendors. However, that's just my opinion, and it might not be the view of anyone else.

Reply:
I understand, any more lights you will shed on me on this subject?